fb_mcbot/views.py

```diff
@@ -34,6 +34,7 @@ def post(self, request, *args, **kwargs):
         for entry in incoming_message['entry']:
             for message in entry['messaging']:
                 if 'message' in message:
-                    answer = AnswerService.process_message(message)
-                    CommunicationService.post_facebook_message(message['sender']['id'], answer)
+                    if (message['sender']['id'] == "1612723942369895") is False:
```

Comment:
That is McBot page's id. Sometimes "message['sender']['id']" is indicating page's (bot's) id instead of user's, and if that id gets passed to the codes, the system will want to create a user for that page id onto the db, but this will cause error when the code is looking for first_name and last_name. This is what I saw when testing locally. That part of the codes acts as a filter.